packages/astro/src/@types/astro.ts

```diff
@@ -1665,6 +1665,20 @@ export interface AstroUserConfig {
 					 * */
 					redirectToDefaultLocale?: boolean;
 
+					/**
+					 * @docs
```

Comment:
I have some questions first here @ematipico 

Is this top-level i18n config? Is this entry in the correct place? Right now, it would be showing up between two `i18n.routing.*` entries.

Shouldn't it also be near `i18n.fallback`? And, this doesn't replace it, but is a separate, same-level configuration? (that's up around line 1567).

I think we should also see an example of both this and fallback configured together in one code sample, since I'm assuming they'd need to work together. Without any `i18n.fallback` configured, you default to showing 404s for pages that don't exist, so you can't have a strategy?